tests/ember-configuration.js

```diff
@@ -99,8 +99,6 @@
     registry.register('adapter:-json-api', DS.JSONAPIAdapter);
     registry.register('serializer:-json-api', DS.JSONAPISerializer);
 
-    registry.injection('serializer', 'store', 'store:main');
```

Comment:
Looks like this line was causing the test to pass previously `registry.injection('serializer', 'store', 'store:main');`